client/payment-methods/index.js

```diff
@@ -285,19 +284,8 @@ const PaymentMethods = () => {
 							status="warning"
 							isDismissible={ false }
 						>
-							{ interpolateComponents( {
-								mixedString: __(
-									'The new WooPayments checkout experience will become the default on October 11, 2023.' +
-										' {{learnMoreLink}}Learn more{{/learnMoreLink}}',
-									'woocommerce-payments'
-								),
-								components: {
-									learnMoreLink: (
-										// eslint-disable-next-line max-len
-										<ExternalLink href="https://woocommerce.com/document/woopayments/payment-methods/additional-payment-methods/#popular-payment-methods" />
-									),
-								},
-							} ) }
+							{ 'You have been switched from the new checkout to your previous checkout experience.' +
+								+'We will keep you posted on the new checkout availability.' }
```

Comment:
I understand this text will be updated soon, but just pointing out the double `+` operator here.

Reply:
Text updated in 9612435aaded660ec6ee6ac611d0c4f5fc80c8bf and reflects the latest decisions.